client/apps/articles_list/client/client.coffee

```diff
@@ -10,7 +10,7 @@ sd = require('sharify').data
 request = require 'superagent'
 icons = -> require('../icons.jade') arguments...
 
-module.exports = ArticlesListView = React.createClass
+module.exports.ArticlesListView = ArticlesListView = React.createClass
```

Comment:
Not quite sure why these were removed @eessex but I added them back into some components. This makes testing a bit easier -- 

```
{ ArticlesListView } = mod = require '... 
mod.__set__ 'SomeOtherComponent', sinon.stub()
React.createElement(ArticlesListView, props)
```
That way we can pass in ArticlesListView to the React rendering side AND modify the requires on the file. Lmk if that makes sense.